ccmlib/scylla_node.py

```diff
@@ -637,7 +637,7 @@ def import_bin_files(self):
                                                  'scylla-jmx'))
 
         parent_dir = os.path.dirname(os.path.realpath(__file__))
-        resources_bin_dir = os.path.join(parent_dir, '..', 'resources', 'bin')
+        resources_bin_dir = os.path.join(parent_dir, 'resources', 'bin')
```

Comment:
how come this works ?

Reply:
The resources folder is inside ccm, moving into ccmlib and making is sure it's part of the installation doesn't change the behavior.